### ISSUE TITLE
Request: send to back for i3 and other basic window managers

### ISSUE THREAD

**Issue by SimLeek** [cheesecakeufo/komorebi#135]
When I run this in i3, it displays the desktop above all of the other windows. This is the same as other desktops, but I think this could be easy enough to implement with something like: if i3, then send to back. If it is, then it seems like it could be worth it.

Otherwise, it works perfectly

**Comment by Twor** [comment]
I also encountered the same problem

**Comment by Adrideux** [comment]
I also have the same problem on i3

**Comment by JacobCZ** [comment]
I have the same problem with Awesome

**Comment by evamvid** [comment]
I have the same problem on i3-gaps

**Comment by paperbenni** [comment]
same problem on dwm

**Comment by person4268** [comment]
same here on i3-gaps; any config that I can use to work around in the meantime?

**Comment by nfrelink** [comment]
Here as well.

**Comment by bondrewd** [comment]
Here too.

**Comment by Aeres-u99** [comment]
same here :-(

**Comment by isovector** [comment]
Also in xmonad

**Comment by abou123** [comment]
Same here

**Comment by camilovietnam** [comment]
Yeah I have this same issue. I really like the animated desktop but I just can't use it with i3

**Comment by alx365** [comment]
Same here

**Comment by Ev1lbl0w** [comment]
@alx365 take a look at #37 for a workaround. This is likely an issue that needs to be fixed on i3wm's side, and not komorebi.

**Comment by isovector** [comment]
It's ridiculous to claim that all of `i3`, `dwm`, `awesome` *and* `xmonad` have the same bug here.

**Comment by Ev1lbl0w** [comment]
That's true, but from a [coding perspective](https://github.com/cheesecakeufo/komorebi/blob/0d2348dacd0cedc374065a0cbbbaf4113fbaa07d/src/OnScreen/BackgroundWindow.vala#L128), komorebi is defining itself as a desktop window, sets itself to below order, and asks to be stickied. As far as the [X documentation](https://specifications.freedesktop.org/wm-spec/wm-spec-1.4.html#STACKINGORDER) goes, it looks like it's doing everything right; and also, I use `bspwm` *(another tilling manager, like the above)*, and it works without any issues.

I will take a better look at this when I've got the time, but like I said, I suspect this might not be an issue with komorebi only.

**Comment by fbievan** [comment]
Any updates?

**Comment by Ev1lbl0w** [comment]
Never got the chance to look it up, and I won't have any from now on.

I recommend opening an issue on i3's repo *(or any other affected WM)* to figure out if it's a problem with komorebi or the WM; you can ping me there if you want so I can give more details if needed

**Comment by fbievan** [comment]
Well, its just that awesome actaully freezes it, after just some looking it seems like dwm, and some others, do not have this function in the code itself, would likely have to patch it in. Just looking at the code, it looks like its just the WM not actaully supporting.

**Comment by fbievan** [comment]
Well it just seems like these window managers do not support the EWMH standard, just from looking at the source code of these, they do not.

**Comment by fbievan** [comment]
This is due to the lack of EWMH compliance in dwm and those other wms
